src/views/Portal/components/ApplicationForm/index.view.tsx

```diff
@@ -82,7 +94,66 @@ const ApplicationForm: React.FC = () => {
 
   const saveData = () => {}
 
-  const submitData = () => {}
+  const submitData = async () => {
+    try {
+      const bodyData = new FormData()
+      bodyData.append("fname", contactFormData.fname)
+      bodyData.append("lname", contactFormData.lname)
+      bodyData.append("email", contactFormData.email || "none")
```

Comment:
It might be better to use the auth0 Hook right here to grab the email in case of some issue with the contactFormData